### PR TITLE
Expanded test data with additional chores and reformatting

### LINKED ISSUE #18: Configure create_tables.py to take cmd line args
Currently, create_tables.py creates the tables with the indexes. We want to add some arguments we can pass in to do different functions, like creating the tables, dropping them, and populating them with test data.

### LINKED ISSUE #20: Refine the test data
Add more depth in dataset

### PR DESCRIPTION
**Pull request by phiphphi**

### CODE REVIEW

**Comment by dishadc** [comment]
Looks like you've added some chores that weren't present in the original data. You also need to update the GetChores tests to include the new chores and make sure the tables are populated with the new test data.
Other than that, everything looks good

**Comment by phiphphi** [comment]
> Looks like you've added some chores that weren't present in the original data. You also need to update the GetChores tests to include the new chores and make sure the tables are populated with the new test data.
> Other than that, everything looks good

Good catch, I ran the tests but forgot to repopulate them with the new data beforehand. I'll fix that up tomorrow.

File: Backend/db/modify_tables.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    "--drop",
+    help="Drops the database tables using the commands in drop-tables.sql.",
+    action="store_true"
+)
+"""
+args = parser.parse_args()
+
+BASEDIR = os.path.abspath(os.path.dirname(".."))
+config = dotenv_values((os.path.join(BASEDIR, ".env")))
+print(BASEDIR)
```
Comment (dishadc) [inline Backend/db/modify_tables.py]: remove the print statement

File: Backend/db/modify_tables.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+server = config["DB_SERVER"]
+database = config["DB_NAME"]
+username = config["DB_USERNAME"]
+password = config["DB_PASSWORD"]
+
+conn = pyodbc.connect(
```
Comment (dishadc) [inline Backend/db/modify_tables.py]: use db/connection.py to create the connection object

**Review by dishadc — CHANGES_REQUESTED** [review]

File: Backend/db/modify_tables.py (hunk 2 above)
Comment (phiphphi) [reply Backend/db/modify_tables.py]: From db.connection import setup_db_conn gives me a ModuleNotFoundError: No module named 'db', do you know a fix for this?

Comment (dishadc) [reply Backend/db/modify_tables.py]: modify_tables.py is in the db directory, so you should only need  `from connection import setup_db_conn`

Comment (phiphphi) [reply Backend/db/modify_tables.py]: My issue then is that connection can't be resolved as an import, Pylance complaining about missing imports. Seems like there's some path issues with importing, as I had to modify my .vscode/settings.json to fix this same issue for app.py, but it still seems to have an issue with this.

Edit: Switching to the backend directory seems to work, but running it in the db directory doesn't (so running db/modify_tables.py works). Weird issue to note.

**Pull request merged**